packages/babel-plugin-transform-es2015-parameters/src/rest.js

```diff
@@ -53,9 +53,18 @@ let memberExpressionOptimisationVisitor = {
       state.deopted = true;
     } else {
       let {parentPath} = path;
+      let grandparentPath = parentPath.parentPath;
```

Comment:
Ha ha, yeah I didn't even think of `grandpa` -- that'd be good for brevity, but I'd probably get in trouble with the internet if I used a gender-specific identifier.